### DISCUSSION TITLE
BUG: Continue Response

### DISCUSSION THREAD

**Discussion by MichaelFomenko** [open-webui/open-webui#3193] [General]
# Bug Report

## Description

**Bug Summary:**
The Continue Response button don't work.

**Steps to Reproduce:**
Start a Conversation and click on the Response Continue Button.

Edit: And I notices a difference in behavior after the first response continue and further chat continue. If i press the continue button on the first, it tries to rewrite the titel of the chat and if i try pressing multiple times fast on the continue button it crashes, or writes an continue with garbage or sometimes it writes good things. And if i press continue on not the first reply in the chat, it sometimes produce the right Response but when i click on continue Response than, it don't continue and if i press multiple times it crashes.

**Expected Behavior:**
The Chat should continue, where it stopped.

**Actual Behavior:**
Don't Continue. But if I press the Button multiple times fast, it starts continuing, but only Gibberish and nonsense.

## Environment

- **Open WebUI Version:** [e.g., v0.3.2]
- **Ollama:** [e.g., 0.1.43]

- **Operating System:** [e.g., Ubuntu 22.04]
- **Browser (if applicable):** [e.g., Chromium]

**Comment by MichaelFomenko** [comment]
Why was my Bug Report moved to this Discussion tab, is this not a Bug?

**Comment by MichaelFomenko** [comment]
???

**Reply by tjbck** [reply]
??????

**Reply by MichaelFomenko** [reply]
?

**Comment by justinh-rahb** [comment]
![1690344054617](https://github.com/open-webui/open-webui/assets/52832301/8afeddce-464d-45d9-88b4-a1c845531f02)

**Reply by MichaelFomenko** [reply]
Who cares if it works on your machine, this Bug is present on other machines.

**Reply by justinh-rahb** [reply]
Can't reproduce on any of my 7 deployed stacks, so it's not present on **any** of my machines, seems like only *yours*.

**Reply by MichaelFomenko** [reply]
Who cares if you can reproduce this Bug on your Machine? This Bug is Present on other Machines.

**Reply by MichaelFomenko** [reply]
And I notices a difference in behavior after the first response continue and further chat continue. If i press the continue button on the first, it tries to rewrite the titel of the chat and if i try pressing multiple times fast on the continue button it crashes, or writes an continue with garbage or sometimes it writes good things. And if i press continue on not the first reply in the chat, it sometimes produce the right Response but when i click on continue Response than, it don't continue and if i press multiple times it crashes.

**Reply by justinh-rahb** [reply]
@MichaelFomenko,

It's important to remember that this is a volunteer-driven project, and we are all here giving our time and effort to make it better. Our contributions are not obligatory, and you are not entitled to support. We are all here because we want to be.

Your issue report was moved because it was deemed to not in fact be a bug. If you disagree with this assertion, it is **YOUR JOB** to tell us why, and bring evidence to the contrary. Your original report even if it bears out to be true is too light on details, this would be closed/moved either way until such is provided.

**Reply by MichaelFomenko** [reply]
This is not the First time, my Bug report was not solved and ignored.

**Reply by justinh-rahb** [reply]
@MichaelFomenko here let me quote this part again since you must have skipped over it:
> **you are not entitled to support**

**Reply by MichaelFomenko** [reply]
OK, I will not report any BUGs here anymore, I will start a Fork of this Project and we will not ignore BUGs in our Project.

**Comment by AlterFalter** [comment]
We have the same issue on all the Claude models for some time now hosted on AWS Bedrock with the [Bedrock Access Gateway](https://github.com/aws-samples/bedrock-access-gateway) as a compatibility layer in between (OpenAI compatible API).
<img width="2850" height="940" alt="{63689F1F-E1C9-438C-9725-C681C48B5762}" src="https://github.com/user-attachments/assets/3c3d1bea-52f3-4c57-9e72-ddab8a33032b" />
This did work for us as well on the previous versions.
Thank you for your support in advance.

**Reply by Classic298** [reply]
that is an upstream provider error, open webui has no hand in that. the api doesnt support assistant prefill so your continue response does not work. this can never have worked in a previous version. you probably tested it on a different model
